RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -85,6 +98,16 @@ private string GetTestModule(IUnitTestSettings settings)
             return formattedModuleTemplate;
         }
 
+        private string GetBinding(bool isEarlyBinding, string theType)
+        {
+            return string.Format(isEarlyBinding ? EarlyBinding : LateBinding, theType);
+        }
+
+        private string DeclareAs(bool isEarlyBinding,string ModuleProvider, string theType)
```

Comment:
Again, `theType` should be `type`, and I'd prefer the enum type to be carried all the way through - not because there's the slightest chance that VBA suddenly allows other binding modes, but because we've abstracted the binding mode in a `BindingMode` enum, so discarding that abstraction just doesn't feel right.

Parameters should be `camelCase`, so `ModuleProvider` should be `moduleProvider`... but `declarationFormat` would be more appropriate.